src/content/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/notice-network-request.mdx

```diff
@@ -0,0 +1,197 @@
+---
+title: noticeNetworkRequest (iOS SDK API)
+type: apiDoc
+shortDescription: Manually record any transactional, HTTP-like network request that completes.
+tags:
+  - Mobile monitoring
+  - New Relic Mobile iOS
+  - iOS SDK API
+metaDescription: New Relic API for iOS mobile app monitoring API to manually record any transactional, HTTP-like network request that completes.
+redirects:
+  - /docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/notice-network-request-ios-sdk-api
+---
+
+## Syntax
+
+```
++ (void)noticeNetworkRequestForURL:(NSURL*)url
+                        httpMethod:(NSString*)httpMethod
+                         withTimer:(NRTimer*)timer
+                   responseHeaders:(NSDictionary*)headers
+                        statusCode:(NSInteger)httpStatusCode
+                         bytesSent:(NSUInteger)bytesSent
+                     bytesReceived:(NSUInteger)bytesReceived
+                      responseData:(NSData *)responseData
+                      traceHeaders:(NSDictionary<NSString*,NSString*>* _Nullable)traceHeaders
+                         andParams:(NSDictionary * _Nullable)params;
+```
+
+Manually record any transactional, HTTP-like network request that completes.
+
+## Requirements
+
+Compatible with all agent versions.
+
+## Description
+
+New Relic will track the URL, response time, status code, and data send/received.
+If the response headers dictionary contains a X-NewRelic-AppData header, New Relic
+will track the association between the mobile app and the web server and
+display the correlation and the server vs. network vs. queue time in the New Relic UI.
+
+If the HTTP status code indicates an error (400 and above) New Relic will also
+track this request as a server error, optionally capturing the response type
```

Comment:
@cdillard-NewRelic Can you explain what "optionally" means? Does it mean the user has to manually configure capturing the response type and encoding the server error?

Reply:
Clarified:

> If the HTTP status code indicates an error (400 and above), New Relic will also track this request as an error. The request header's dictionary and the response body data will be encoded as a server error in the New Relic UI.